obal/data/module_utils/chdir.py

```diff
@@ -0,0 +1,19 @@
+"""
+A chdir context manager
+"""
+import os
+from contextlib import contextmanager
+
+
+@contextmanager
+def chdir(directory):
```

Comment:
shouldn't we just put that into `obal/data/module_utils/obal.py`? new file for each helper seems a bit too much?

Reply:
Deal.

Reply:
Next update will remove this file.